osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/application/threads/RecoverKeyProcessTest.java

```diff
@@ -135,9 +140,6 @@ public void testWhenConnectionFailedThenConnectionClosedAtThrottlingService() th
     // GIVEN
     when(this.domainHelperService.findDlmsDevice(DEVICE_IDENTIFICATION, IP_ADDRESS))
         .thenReturn(DEVICE);
-    when(this.secretManagementService.hasNewSecretOfType(
-            MESSAGE_METADATA, DEVICE_IDENTIFICATION, E_METER_AUTHENTICATION))
-        .thenReturn(true);
     when(this.hls5Connector.connectUnchecked(any(), any(), any(), any())).thenReturn(null);
 
     // WHEN
```

Comment:
this.secretManagementService can be removed from `inOrder(this.throttlingService, this.hls5Connector, this.secretManagementService)`